### PR TITLE
chore: update Extism namespace to `extism:host/env`

### PR DESCRIPTION
**Pull request by zshipko**
Similar to: https://github.com/extism/extism/pull/511

- Updates the SDK to be compatible with new namespaces
- Updates test Wasm to use the new namespaces
- Some updates to our WASI code for better reactor support. After updating some of the plugins I ran into issues with how the initialization was happening for non-command modules. 
- Uses `GET` by default if no method is passed to `http_request`

### CODE REVIEW

File: src/call-context.ts
Comment (chrisdickinson) [inline src/call-context.ts:274]:
```suggestion
    http_request: (_requestOffset: bigint, _bodyOffset: bigint): bigint => {
```

**Review by chrisdickinson — APPROVED** [review]
Thanks for sorting out the Haskell/WASI init order & updating the namespaces!

I saw one bug – `ttp_request` instead of `http_request` – and a formatting thing in the polyfills, but `just format` should fix that up. Otherwise this LGTM! Once these are fixed feel free to merge at will!

**Comment by nilslice** [comment]
Added #26 to run tests, just case something is weird. `just test` failed to parse `deno.lock` on my machine. I probably skipped a step, so I'm just running CI instead :smile:

**Comment by chrisdickinson** [comment]
@nilslice Ah yeah – you might check your Deno version – v1.36 is the minimum supported version. Thanks

**Comment by nilslice** [comment]
@chrisdickinson, indeed I had too old of a Deno installed!

**Pull request merged**
